Forge.Forms/src/Forge.Forms.Tests/Properties/AssemblyInfo.cs

```diff
@@ -31,5 +31,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
+[assembly: AssemblyVersion("2020.11.25.3")]
+[assembly: AssemblyFileVersion("2020.11.25.3")]
```

Comment:
This could cause some nasty breaks for library consumers.

Reply:
This is a standard setting on my side, ignore it please!